### PR TITLE
fix(tour): findDomNode throw warning in strict mode

### LINKED ISSUE #39921: Tour在StrictMode模式下卡片指向元素错误
### Reproduction link

[https://8prkit.csb.app/](https://8prkit.csb.app/)

### Steps to reproduce

该示例为官方文档示例源码 请点击后查看控制台！

### What is expected?

去除warning 在实际开发中会遇到不可预期的错误

### What is actually happening?

Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Trigger which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here: https://reactjs.org/link/strict-mode-find-node
    at div
    at eval (https://8prkit.csb.app/node_modules/rc-trigger/es/Popup/index.js:26:22)
    at eval (https://8prkit.csb.app/node_modules/rc-util/es/Portal.js:11:25)
    at Trigger (https://8prkit.csb.app/node_modules/rc-trigger/es/index.js:62:35)
    at Tour (https://8prkit.csb.app/node_modules/@rc-component/tour/es/Tour.js:47:32)
    at Tour (https://8prkit.csb.app/node_modules/antd/es/tour/index.js:27:21)
    at div
    at App (https://8prkit.csb.app/index.js:25:34)

| Environment | Info |
| --- | --- |
| antd | 5.1.1 |
| React | 18.2 |
| System | win11 |
| Browser | Edge Chorme |

---
codesandbox无法复现在StrictMode下实际场景中的卡片指向元素错误 但是控制台的warning可以看到

### PR DESCRIPTION
**Pull request by kiner-tang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [X] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

- fix: https://github.com/ant-design/ant-design/issues/39921

### 💡 Background and solution

- fix: findDomNode throw warning in strict mode

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fixed the findDomNode method warning thrown by the tour component when called in strict mode |
| 🇨🇳 Chinese | 修复`tour`组件在严格模式下调用`findDomNode`抛出警告问题 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
changelog 应该包含 `@rc-component/tour` 1.1.0-1.5.0所有变化

**Comment by yoyo837** [comment]
不太对劲, `feature` 分支已经更新到1.4.0了, 你这个1.5.0直接发master上面合适吗?

File: components/tour/demo/basic.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ const App: React.FC = () => {
   ];
 
   return (
-    <>
+    <React.StrictMode>
```
Comment (zombieJ) [inline components/tour/demo/basic.tsx:38]: 这个应该不用的，每个 Demo 最外面都抱着 StrictMode

Comment (kiner-tang) [reply components/tour/demo/basic.tsx:38]: ok

**Review by zombieJ — APPROVED** [review]

**Comment by afc163** [comment]
这个应该发 feature 把，不然 https://github.com/ant-design/ant-design/pull/39980 就错乱了。

**Comment by yoyo837** [comment]
我也发了疑问, 但豆酱 Approve, 我就没说啥了

**Comment by afc163** [comment]
我回滚， @kiner-tang 重新发个到 feature 分支。

**Comment by kiner-tang** [comment]
> 我回滚， @kiner-tang 重新发个到 feature 分支。

好的

**Pull request merged**